client/run-cypress-tests.sh

```diff
@@ -4,7 +4,10 @@ export ARLO_SMTP_HOST=localhost
 export ARLO_SMTP_PORT=1025
 export ARLO_SMTP_USERNAME=cypress-smtp-username
 export ARLO_SMTP_PASSWORD=cypress-smtp-password
-export ARLO_FILE_UPLOAD_STORAGE_PATH=s3://arlo-file-uploads-dev
+if [[ -n $AWS_ACCESS_KEY_ID ]] && [[ -n $AWS_SECRET_ACCESS_KEY ]]
+then
+    export ARLO_FILE_UPLOAD_STORAGE_PATH=s3://arlo-file-uploads-dev
+fi
```

Comment:
This change seems unrelated to this PR. Can you remove it? Feel free to make a separate PR for it if you'd like

Reply:
Ah yes, unfortunately this change is necessary for me to pass the cypress tests since my build pipeline account doesn't have any aws access key env vars set (I assume this is why the build fails for me but not for others working on the project?).

I've got another PR over [here](https://github.com/votingworks/arlo/pull/1696) that includes these changes, once it gets merged in I'll fix this one : )

Reply:
Oh interesting. Ok, sounds good to wait for the other PR!